scene/3d/node_3d.cpp

```diff
@@ -703,7 +703,7 @@ void Node3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_scale"), &Node3D::get_scale);
 	ClassDB::bind_method(D_METHOD("set_global_transform", "global"), &Node3D::set_global_transform);
 	ClassDB::bind_method(D_METHOD("get_global_transform"), &Node3D::get_global_transform);
-	ClassDB::bind_method(D_METHOD("get_parent_spatial"), &Node3D::get_parent_spatial);
+	ClassDB::bind_method(D_METHOD("get_parent_node3d"), &Node3D::get_parent_node3d);
```

Comment:
In theory this might have to be `get_parent_node_3d` to properly respect snake case... but it looks weird given the potential confusion with `Node`.